app.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+(function() {
+  angular
+  .module("toDo", [])
+  .controller("toDoController", toDoController);
+
+  function toDoController() {
+    this.data = [
+      {title: "take out ethe trash", completed: false},
+      {title: "walk the dog", completed: false},
+      {title: "learn angular", completed: false},
+      {title: "buy groceries", completed: true}
+    ];
+
+    this.create = function(){
+      this.data.push({title: this.newTask, completed: false});
+      this.newTask = {} //Removes text from input field
+    }
+
+    this.edit = function() {
+
+    }
+
+    this.delete = function() {
+      this.data.pop()
+    }
```

Comment:
You're not always going to want to remove the last thing in your data array, so `.pop` isn't the right move here. You'll need to make sure of an index value so that you can remove a particular item from `vm.data`.